.mailmap

```diff
@@ -8,6 +8,7 @@
 #
 Aaron Bassett <abassett@gmail.com>
 Abhishek Lekshmanan <abhishek.lekshmanan@ril.com> <abhishek.lekshmanan@gmail.com>
+Abhishek Lekshmanan <abhishek@suse.com> <alekshmanan@suse.com>
```

Comment:
@dachary - Why to update peoplemap? mailmap updated with both email ids...do I miss some thing here?